addons/captives/XEH_postInit.sqf

```diff
@@ -35,6 +35,18 @@ if (isServer) then {
 
 if (!hasInterface) exitWith {};
 
+    //Add Keybind:
+["ACE3 Common", QGVAR(captives), [(localize LSTRING(KeyComb)), (localize LSTRING(KeyComb_description))],
+{
+    if !(cursorObject isKindOf "CAManBase" || {(cursorObject distance player) > getNumber (configFile >> "CfgVehicles" >> "CAManBase" >> "ACE_Actions" >> "ACE_ApplyHandcuffs" >> "distance")}) exitWith {};
```

Comment:
use `ace_player` not player

this code should also return a bool, generally false if nothing happens, and true if it will
so false if out of range, true if we run `setHandcuffed` and false at the bottom

Reply:
also add
```
if !([ACE_player, cursorObject, []] call EFUNC(common,canInteractWith)) exitWith {false};
```